### PR TITLE
docs: update algorithmiq-tem notebook

### PR DESCRIPTION
**Pull request by robertodr**

### CODE REVIEW

File: docs/guides/algorithmiq-tem.ipynb [hunk 1]
```diff
@@ -261,13 +262,15 @@
     "\n",
     "Name | Choices | Description  | Default\n",
     "-- | -- | -- | --\n",
-    "`max_bond_dimension` | int | The maximum bond dimension to be used for the tensor networks. | 500 |\n",
+    "`tem_max_bond_dimension` | int | The maximum bond dimension to be used for the tensor networks. | 500 |\n",
```
Comment (Eric-Arellano) [inline docs/guides/algorithmiq-tem.ipynb:265]: Did the name change, or `max_bond_dimension` was always wrong?

File: docs/guides/algorithmiq-tem.ipynb [hunk 2]
```diff
@@ -245,9 +243,12 @@
     "`options` | dict | Input options. See `Options` section for more details. | No | See `Options` section for more details.| \\{\"max_bond_dimension\": 100\\}\n",
     "\n",
     "\n",
-    "<Admonition type=\"caution\"> TEM does not currently support parametrized circuits. The `parameters` argument should be set to `None` if `precision` is specified.</Admonition>\n",
+    "<Admonition type=\"caution\">TEM currently has the following limitations:\n",
     "\n",
-    "<Admonition type=\"caution\"> TEM currently supports circuits with linear connectivity. This restriction will be removed in future versions.</Admonition>"
+    "    - Parametrized circuits are not supported. The parameters argument should be set to None if precision is specified.\n",
```
Comment (Eric-Arellano) [inline docs/guides/algorithmiq-tem.ipynb:248]:
```suggestion
    "    - Parametrized circuits are not supported. The parameters argument should be set to `None` if precision is specified.\n",
```

**Review by Eric-Arellano — APPROVED** [review]

File: docs/guides/algorithmiq-tem.ipynb (hunk 1 above)
Comment (robertodr) [reply docs/guides/algorithmiq-tem.ipynb:265]: I think it was correct initially, then was changed, and not updated in the notebook.

**Comment by robertodr** [comment]
I have no idea how to fix the lint...

**Comment by Eric-Arellano** [comment]
@robertodr there are two issues:

- Too long of line. We run Ruff to make the code blocks shorter so they don't have horizontal scrolling
- Issue with the `<ul>` you set up

You can use this diff to fix things. FYI you can save the diff in a file like `review-changes.patch`, then run `git apply review-changes.patch`.

```diff
diff --git a/docs/guides/algorithmiq-tem.ipynb b/docs/guides/algorithmiq-tem.ipynb
index 95369207cd..dbbaf96036 100644
--- a/docs/guides/algorithmiq-tem.ipynb
+++ b/docs/guides/algorithmiq-tem.ipynb
@@ -177,7 +177,9 @@
     "pub = (qc, [observable])\n",
     "options = {\"default_precision\": 0.02}\n",
     "\n",
-    "job = tem.run(pubs=[pub], instance=instance, backend_name=backend_name, options=options)"
+    "job = tem.run(\n",
+    "    pubs=[pub], instance=instance, backend_name=backend_name, options=options\n",
+    ")"
    ]
   },
   {
@@ -244,13 +246,12 @@
     "\n",
     "\n",
     "<Admonition type=\"caution\">\n",
-    "TEM currently has the following limitations:\n",
-    "<ul>\n",
-    "  <li>Parametrized circuits are not supported. The parameters argument should be set to <tt>None</tt> if precision is specified.</li>\n",
-    "  <li>Only circuits with linear connectivity are supported. This restriction will be removed in future versions.</li>\n",
-    "  <li>Non-unitary gates, such as reset, measure, and all forms of control flow are not supported. Support for reset will be added in upcoming releases.</li>\n",
-    "</ul> \n",
-    "</Admonition>\n"
+    "  TEM currently has the following limitations:\n",
+    "\n",
+    "  - Parametrized circuits are not supported. The parameters argument should be set to `None` if precision is specified.\n",
+    "  - Only circuits with linear connectivity are supported. This restriction will be removed in future versions.\n",
+    "  - Non-unitary gates, such as reset, measure, and all forms of control flow are not supported. Support for reset will be added in upcoming releases.\n",
+    "</Admonition>"
    ]
   },
   {
```

The warning will render like this, which you can preview by running `./start` in this repo.

Screenshot 2024-11-12 at 9 16 05 AM

**Comment by Eric-Arellano** [comment]
Thanks for the fix! We will do a deploy later today to docs.quantum.ibm.com.

**Pull request merged**